### DISCUSSION TITLE
Database: docker image vs code output

### DISCUSSION THREAD

**Discussion by jklejdysz** [open-discourse/open-discourse#114] [Q&A]
Hi, is the docker image of the database always up-to-date with the datasets generated from the ground up using the code in the dev branch?

**Comment by JabobKrauskopf** [comment]
Hi! Thanks for the question.

The database in the Docker image only contains the plenary sessions up to the time of the upload.
If you generate the database from the ground up, it'll contain all the plenary minutes up until that point.
That is why we are working on #56 in order to re-create and upload the database every week and include recent plenary sessions.

Hope this helps :)
